lib/stockboy/configurator.rb

```diff
@@ -213,5 +218,24 @@ def wrap_provider(config)
       config[:provider] = ProviderRepeater.new(config[:provider], &repeat)
     end
 
+    # Store for template variables to be used in the DSL.
+    #
+    # A "frozen" copy is made of any values passed, such that template variables are
+    # locked at job definition time. To use a different set of variables, the job must
+    # be re-defined.
+    #
+    class EnvVars
```

Comment:
We could provide access to a few hash reader methods like `key?` and `keys` for convenience. (Can add later.)